osu! Mapping Olympiad #7: Results (osu!mania) (#4464)
ppy/osu-wiki: news/2020-11-15-osu-mapping-olympiad-7-concludes.md

---
layout: post
title: "osu! Mapping Olympiad #7: Results (osu!mania)"
date: 2020-11-15 18:00:00 +0000
---

> Here come the results of the 2nd edition of the osu!mania Mapping Olympiads, including sick surprises for sure!

Reviewer: ```suggestion
Here come the results of the 2nd edition of the osu!mania Mapping Olympiad, including sick surprises for sure!
```

![](/wiki/shared/news/banners/osu_mapping_olympiad.jpg)

> For the first time in the history of osu!mania Mapping Olympiad, two Olympians have received the same score, hence shared the first place! Please give a huge round of applause to **[Kawawa](https://osu.ppy.sh/users/4647754)** and **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**!

Reviewer: ```suggestion
For the first time in the history of the osu!mania Mapping Olympiad, two Olympians have received the same score and share first place! Please give a huge round of applause to **[Kawawa](https://osu.ppy.sh/users/4647754)** and **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**!
```

> During this iteration, contestants had to create a full mapset on one of 5 songs from several artists, featuring [PUP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN](https://osu.ppy.sh/beatmaps/artists/33), [IAHN](https://osu.ppy.sh/beatmaps/artists/3), [Frums](https://osu.ppy.sh/beatmaps/artists/68) and [Cranky](https://osu.ppy.sh/beatmaps/artists/23)! You can find more details about this contest's criteria over [here](https://osu.ppy.sh/community/contests/97)!

Reviewer: ```suggestion
During this iteration, contestants had to create a full mapset for 1 of 5 songs from several Featured Artists, including [PUP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN](https://osu.ppy.sh/beatmaps/artists/33), [IAHN](https://osu.ppy.sh/beatmaps/artists/3), [Frums](https://osu.ppy.sh/beatmaps/artists/68) and [Cranky](https://osu.ppy.sh/beatmaps/artists/23)! You can find more details about this contest's criteria over [here](https://osu.ppy.sh/community/contests/97)!
```

The leaderboards of the 10 best entries are shown right below!

> | Rank | Username | Entry Name | Rankability (45) | Song Interpretation (60) | Originality (45) | Hitsounding (15) | Judge Impression (30) | Total Score |

Reviewer: the "entry name" column is useless. better to delete it or replace it with the song name/map link

Author: i prefer to remove it, it would look weird having the song name imo

|------|---------------|----------------------|------------------------|-----------------|----------------|-----------------|------------------|-------------|
| #1   | Kawawa   | Moonflower        | 41.5                     | 42            | 27             | 13              | 17             | **140.5**     |
| #1   | Toaph Daddy   | Zinnia        | 33                     | 46            | 30.5             | 10              | 21             | **140.5**     |
| #3   | lenpai       | Lupine         | 35                     | 39              | 36           | 12            | 16               | **138**     |
| #3   | BringoBrango       | Azalea         | 18                     | 49.5              | 34.5           | 13            | 23               | **138**     |
| #5   | Garalulu | Lily       | 14                   | 41.5            | 30             | 12            | 18               | **115.5**   |

> | #6   | chouyaaa      | Hepatica     | 27.5                   | 31            | 19.5           | 13.5            | 13.5             | **105**   |

Reviewer 2: chouyaa username should be [Mirea](https://osu.ppy.sh/users/8404646)

| #7 | EimitMaggi999 | Iris | 31 | 27.5 | 18 | 13.5 | 12.5 | **102.5** |
| #8 | Noch Einen | Tulip | 35 | 26 | 21 | 8.5 | 11.5 | **102** |
| #9 | fktang | Buttercup | 33 | 29 | 14 | 14.5 | 10 | **100.5** |
| #10 | FAMoss | Celosia | 36.5 | 22 | 14 | 14.3 | 7.5 | **94.3** |

Congratulations to **[Kawawa](https://osu.ppy.sh/users/4647754)** and **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**, our two winners! They both managed to get an impressive score of **140.5** despite having quite different scores in each category.

> After a second place on their last Olympiad participation, **[Kawawa](https://osu.ppy.sh/users/4647754)** ties **[Sinnoh](https://osu.ppy.sh/users/4236057)** for the first place on the Olympian points leaderboards! Their score on the **Rankability** category clearly shows their ability to provide high-quality content while being original on the side.

Reviewer: link to the "Olympian points leaderboards" here. it's hard to see how sinnoh is relevant until you see the leaderboard

> | #10   | FAMoss   | Celosia     | 36.5                     | 22            | 14           | 14.3            | 7.5             | **94.3**     |

Reviewer 3: ```suggestion
| Rank | Username | Entry Name | Rankability (45) | Song Interpretation (60) | Originality (45) | Hitsounding (15) | Judge Impression (30) | Total Score |
| :-- | :-- | :-- | :-: | :-: | :-: | :-: | :-: | :-: |
| #1 | Kawawa | Moonflower | 41.5 | 42 | 27 | 13 | 17 | **140.5** |
| #1 | Toaph Daddy | Zinnia | 33 | 46 | 30.5 | 10 | 21 | **140.5** |
| #3 | lenpai | Lupine | 35 | 39 | 36 | 12 | 16 | **138** |
| #3 | BringoBrango | Azalea | 18 | 49.5 | 34.5 | 13 | 23 | **138** |
| #5 | Garalulu | Lily | 14 | 41.5 | 30 | 12 | 18 | **115.5** |
| #6 | chouyaaa | Hepatica | 27.5 | 31 | 19.5 | 13.5 | 13.5 | **105** |
| #7 | EimitMaggi999 | Iris | 31 | 27.5 | 18 | 13.5 | 12.5 | **102.5** |
| #8 | Noch Einen | Tulip | 35 | 26 | 21 | 8.5 | 11.5 | **102** |
| #9 | fktang | Buttercup | 33 | 29 | 14 | 14.5 | 10 | **100.5** |
| #10 | FAMoss | Celosia | 36.5 | 22 | 14 | 14.3 | 7.5 | **94.3** |
```
since we do not beautify the tables, correct formatting should be something like this

Author: Applied

> New to this contest, but an osu!mania veteran, **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**, joins **[Kawawa](https://osu.ppy.sh/users/4647754)**'s step on the first place! They impressed the judges **[Unpredictable](https://osu.ppy.sh/users/7560872)** and **[Kamuy](https://osu.ppy.sh/users/7439226)** with their song interpretation and received an outstanding score of 18 out of 20 on the **Song Interpretation** category!

Reviewer: ```suggestion
osu!mania veteran **[Toaph Daddy](https://osu.ppy.sh/users/7616811)** joins **[Kawawa](https://osu.ppy.sh/users/4647754)**'s step on the first place! They impressed the judges **[Unpredictable](https://osu.ppy.sh/users/7560872)** and **[Kamuy](https://osu.ppy.sh/users/7439226)** with their song interpretation and received an outstanding score of 18 out of 20 on the **Song Interpretation** category!
```
too many commas close together makes reading the first sentence suck

> Congratulations to [**Kawawa**](https://osu.ppy.sh/users/4647754) and [**Toaph Daddy**](https://osu.ppy.sh/users/7616811), our two winners! They both managed to get an impressive score of **140.5** despite having quite different scores in each categories. 

Reviewer 3: ```suggestion
Congratulations to **[Kawawa](https://osu.ppy.sh/users/4647754)** and **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**, our two winners! They both managed to get an impressive score of **140.5** despite having quite different scores in each category. 
```

> Let's jump on another spicy tie for the third place! **[lenpai](https://osu.ppy.sh/users/5314573)** and **[BringoBrango](https://osu.ppy.sh/users/10274043)** both reached the third place after receiving a score of **138** which is not that far from first place's score! They both scored an excellent score in the Originality category, hitting the highest scores in that category among all the entries!

Reviewer: ```suggestion
Let's jump on another spicy tie for the third place! **[lenpai](https://osu.ppy.sh/users/5314573)** and **[BringoBrango](https://osu.ppy.sh/users/10274043)** both reached the third place after receiving a score of **138** which is not that far from first place's score! They both achieved an excellent score in the Originality category, hitting the highest scores in that category among all the entries!
```
"scored a score" sounds dumb

> After a second place on their last Olympiad participation, [**Kawawa**](https://osu.ppy.sh/users/4647754) ties [**Sinnoh**](https://osu.ppy.sh/users/4236057) for the first place on the Olympian points leaderboards! Their score on the **Rankability** category clearly shows their ability to provide high-quality content while being original on the side. 

Reviewer 3: ```suggestion
After a second-place on their last Olympiad participation, **[Kawawa](https://osu.ppy.sh/users/4647754)** ties **[Sinnoh](https://osu.ppy.sh/users/4236057)** for the first place on the Olympian points leaderboards! Their score on the **Rankability** category clearly shows their ability to provide high-quality content while being original on the side. 
```

Reviewer 4: Would not hyphenate "second place" unless an additional noun is present for the composite adjective to work (i.e. second-place finish)

Reviewer 3: ah, my bad. turns out having Grammarly apply a second check was not a good idea

>  Please take a look at the comments from the judges along with the full results for all entries, right [here](https://docs.google.com/spreadsheets/d/11CNN7iYipBwffVCFJ8ulPC8CounECRPp0nc-MmhBwQk/edit?usp=sharing)!

Reviewer: ```suggestion
 Please take a look at the comments from the judges along with the full results for all entries right [here](https://docs.google.com/spreadsheets/d/11CNN7iYipBwffVCFJ8ulPC8CounECRPp0nc-MmhBwQk/edit?usp=sharing)!
```

> New into this contest, but a veteran in osu!mania, [**Toaph Daddy**](https://osu.ppy.sh/users/7616811) joins [**Kawawa**](https://osu.ppy.sh/users/4647754)'s step on the first place! They impressed the judges [**Unpredictable**](https://osu.ppy.sh/users/7560872) and [**Kamuy**](https://osu.ppy.sh/users/7439226) with their song interpretation and receive an outstanding score of 18 out of 20 on the **Song Interpretation** category! 

Reviewer 3: ```suggestion
New to this contest, but an osu!mania veteran, **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**, joins **[Kawawa](https://osu.ppy.sh/users/4647754)**'s step on the first place! They impressed the judges **[Unpredictable](https://osu.ppy.sh/users/7560872)** and **[Kamuy](https://osu.ppy.sh/users/7439226)** with their song interpretation and received an outstanding score of 18 out of 20 on the **Song Interpretation** category! 
```

We would like to thank everyone who took part in this contest, and congratulations to the top three podium finishers! The points and rewards attribution will be done as shown below:

> - **First place**: 4 months of osu!supporter, 6 Olympiad Points (resulting in 'Mapping Olympian' badge), the mapset pushed to Ranked section, and the **Elite Mapper** user title.

Reviewer: ```suggestion
- **First place**: 4 months of osu!supporter, 6 Olympiad Points (resulting in *Mapping Olympian* badge), the mapset pushed to Ranked section, and the **Elite Mapper** user title.
```
looks better than singlequotes

> Let's jump on another spicy tie for the third place! [**lenpai**](https://osu.ppy.sh/users/5314573) and [**BringoBrango**](https://osu.ppy.sh/users/10274043) both reach third place after they receive a score of **138** which is not that far from first place's score!  They both scored an excellent score in the Originality category, hitting the highest scores in that category among all the entries! 

Reviewer 3: ```suggestion
Let's jump on another spicy tie for the third place! **[lenpai](https://osu.ppy.sh/users/5314573)** and **[BringoBrango](https://osu.ppy.sh/users/10274043)** both reached the third place after receiving a score of **138** which is not that far from first place's score! They both scored an excellent score in the Originality category, hitting the highest scores in that category among all the entries! 
```

>  Please take a look at the comments from the judges along with the full results for all entries, right [here!](https://docs.google.com/spreadsheets/d/11CNN7iYipBwffVCFJ8ulPC8CounECRPp0nc-MmhBwQk/edit?usp=sharing)

Reviewer 3: ```suggestion
 Please take a look at the comments from the judges along with the full results for all entries, right [here](https://docs.google.com/spreadsheets/d/11CNN7iYipBwffVCFJ8ulPC8CounECRPp0nc-MmhBwQk/edit?usp=sharing)!
```

- **Third place**: 2 months of osu!supporter, 2 Olympiad Points.

A complete listing of all accumulated Olympiad Points can be located [here](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing). Updates to the listing will be posted shortly after the conclusion of each contest.

> All of this wouldn't be possible without the immense help of our three judges: **[Hydria](https://osu.ppy.sh/users/808176)**, **[Kamuy](https://osu.ppy.sh/users/7439226)**, and **[Unpredictable](https://osu.ppy.sh/users/7560872)** all of whom have also helped a lot during the early organization of this iteration. Thank you a bunch!

Reviewer: ```suggestion
All of this wouldn't be possible without the immense help of our three judges: **[Hydria](https://osu.ppy.sh/users/808176)**, **[Kamuy](https://osu.ppy.sh/users/7439226)**, and **[Unpredictable](https://osu.ppy.sh/users/7560872)**, all of whom have also helped a lot during the early organization of this iteration. Thank you a bunch!
```

> * **First place**: 4 months of osu!supporter, 6 Olympiad Points (resulting in 'Mapping Olympian' badge), the mapset pushed to Ranked section and the Elite Mapper title

Reviewer 3: ```suggestion
- **First place**: 4 months of osu!supporter, 6 Olympiad Points (resulting in 'Mapping Olympian' badge), the mapset pushed to Ranked section, and the **Elite Mapper** user title.
```

> * **Third place**: 2 months of osu!supporter, 2 Olympiad Points

Reviewer 3: ```suggestion
- **Third place**: 2 months of osu!supporter, 2 Olympiad Points.
```

> [A complete listing of all accumulated Olympiad Points can be located here.](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing) Updates to the listing will be posted shortly after the conclusion of each contest.

Reviewer 3: ```suggestion
A complete listing of all accumulated Olympiad Points can be located [here](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing). Updates to the listing will be posted shortly after the conclusion of each contest.
```
either way is fine, but punctuation should be left out of the hyperlinks

> All of this wouldn't be possible without the immense help of our three judges: [**Hydria**](https://osu.ppy.sh/users/808176), [**Kamuy**](https://osu.ppy.sh/users/7439226) and [**Unpredictable**](https://osu.ppy.sh/users/7560872) who also helped a lot in the early organization of this contest iteration. Thank you a bunch!

Reviewer 3: ```suggestion
All of this wouldn't be possible without the immense help of our three judges: **[Hydria](https://osu.ppy.sh/users/808176)**, **[Kamuy](https://osu.ppy.sh/users/7439226)**, and **[Unpredictable](https://osu.ppy.sh/users/7560872)** all of whom have also helped a lot during the early organization of this iteration. Thank you a bunch!
```

Curious about the entries? Find them all in one place right [here](https://drive.google.com/file/d/1ulTk_EVRKf8c-DMHqW4I0FwCB5RbJfL_/view)!

> We sincerely hope you enjoyed this iteration of the osu! Mapping Olympiad as much as we did! This is surely not the last and the contest will be back soon with new concepts to make them even more creative and interesting for you! Congratulations once again to [**Kawawa**](https://osu.ppy.sh/users/4647754) and [**Toaph Daddy**](https://osu.ppy.sh/users/7616811) for your victory!  

Reviewer 3: ```suggestion
We sincerely hope you enjoyed this iteration of the osu! Mapping Olympiad as much as we did! This is surely not the last and the contest will be back soon with new concepts to make them even more creative and interesting for you! Congratulations once again to **[Kawawa](https://osu.ppy.sh/users/4647754)** and **[Toaph Daddy](https://osu.ppy.sh/users/7616811)** for your victory!  
```
